test/gtest/uct/test_md.cc

```diff
@@ -292,7 +292,9 @@ UCS_TEST_P(test_md, mem_type_detect_mds) {
         if (status == UCS_ERR_UNSUPPORTED) {
             UCS_TEST_MESSAGE << ucs_memory_type_names[alloc_mem_type] << ": "
                              << "unsupported mem_query";
-            /* don't continue with this memory type */
+            /* don't continue with this memory type.
+             * ROCM md has limitation on memory type
+             * detection - test is failed on ROCM_MANAGED memory type */
```

Comment:
maybe we can remove UCS_MEMORY_TYPE_ROCM_MANAGED from uct_rocm_copy_md_query for now?
then it will skip the test in line 278

Reply:
@souravzzz any idea why rocm is not able to detect ROCM_MANAGED (after allocated by hipMallocManaged) in mem_buffer.cc?

Reply:
yep, good point, removed UCS_MEMORY_TYPE_ROCM_MANAGED from caps

Reply:
@yosefe Managed memory is not yet fully supported by the HIP API. It is able to allocate managed memory through `hipMallocManaged()` but does not set the necessary flag to differentiate it from regular GPU memory. This is going to be fixed in a later ROCm release.

Reply:
so i guess it's ok to remove the flag for CUDA_MANAGED detection in this PR?